src/coprocessor/endpoint.rs

```diff
@@ -284,6 +286,7 @@ impl<E: Engine> Endpoint<E> {
                         snap,
                         req_ctx,
                         quota_limiter,
+                        is_auto,
```

Comment:
no need to pass it, we can deduce it in `AnalyzeContext::new`

Reply:
> no need to pass it, we can deduce it in `AnalyzeContext::new`

Good suggestion. fixed.